src/modules/order/domains/entities/order.entity.ts

```diff
@@ -79,4 +91,7 @@ export class OrderEntity extends AbstractEntity {
   @OneToOne(() => PaymentEntity, (payment) => payment.id)
   @JoinColumn()
   payment: PaymentEntity;
+
+  @ManyToOne(() => UserEntity, (user) => user.orders)
+  user: UserEntity;
 }
```

Comment:
consider adding
- [ ] products -> ManyToOne, from which relation lessor can get their order.
- [ ] isAccepted -> allow lessor to reject an order

another thing to keep in mind: instead of payment module, which stores user credit card, we can add a wallet module, which store their transaction info (their billing, money) using a third party (MoMo testing platform is preferable), this wallet module can store up an amount of money, from which user can withdraw to their third party account